portfolio.html

```diff
@@ -11,6 +11,7 @@
 <body>
   <div id="image-container">
     <img id="ellipses" src="./images/Top-circles.png" alt="circles">
+    <img id="background" src="./images/portfolio-background-shapes-mobile@2x.png" alt="background">
   </div>
   <header>
     <nav>
```

Comment:
[Optional] You seem to have your header fixed and that causes it to overlap with other content when a scroll is made. I suggest that you correct that so that it ceases to overlap.